Lib CPP template: Ubuntu/build WF fails

**Associated Template/Command/Core**

<!-- State the template handle or command. -->
Template: Lib-CPP

**Describe the bug**

The Ubuntu/build WF fails (in the `configure` step) with:
```
Run cmake -Bbuild -DCMAKE_INSTALL_PREFIX=$GITHUB_WORKSPACE/$INSTALL_LOCATION -D{ cookiecutter.project_slug }_ENABLE_CODE_COVERAGE=1
CMake Error: Parse error in command line argument: -D{
Should be: VAR:type=value

CMake Error: Problem processing arguments. Aborting.
```


**To Reproduce**

Steps to reproduce the behavior:
1. Run COOKIETEMPLE with command: `create`
2. Select: `lib cpp`
3. Push to Github Repo
4. See failing WF

**Expected behavior**

<!-- A clear and concise description of what you expected to happen. -->

Workflow should pass at initial commit! (Other WF are passing)

--- a/cookietemple/create/templates/lib/lib_cpp/{{ cookiecutter.project_slug }}/.github/workflows/build_linux.yml
+++ b/cookietemple/create/templates/lib/lib_cpp/{{ cookiecutter.project_slug }}/.github/workflows/build_linux.yml
@@ -38,7 +38,7 @@ jobs:
         cd ../{{ cookiecutter.project_slug }}
 
     - name: configure
-      run: cmake -Bbuild -DCMAKE_INSTALL_PREFIX=$GITHUB_WORKSPACE/$INSTALL_LOCATION -D{ cookiecutter.project_slug }_ENABLE_CODE_COVERAGE=1
+      run: cmake -Bbuild -DCMAKE_INSTALL_PREFIX=$GITHUB_WORKSPACE/$INSTALL_LOCATION -D{{ cookiecutter.project_slug }}_ENABLE_CODE_COVERAGE=1
       
     - name: build
       run: cmake --build build --config $BUILD_TYPE -j4

